Cargo.toml

```diff
@@ -12,7 +12,7 @@ readme = "README.md"
 license = "MIT"
 
 [dependencies]
-bdk-macros = "0.2"
+bdk-macros = { path = "./macros" }
```

Comment:
Is there a better way to do this?

Reply:
I think this is the best way to do it, and will be updated to an actual version when this PR is included in a release.